packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx

```diff
@@ -85,4 +86,9 @@ BreadcrumbDivider.defaultProps = {
   accessibility: breadcrumbDividerBehavior,
 };
 
-BreadcrumbDivider.propTypes = commonPropTypes.createCommon();
+BreadcrumbDivider.propTypes = {
+  ...commonPropTypes.createCommon({
+    content: false,
+  }),
+  content: customPropTypes.shorthandAllowingChildren,
+};
```

Comment:
This change is not valid. `children` and `content` cannot be used in the same time

From which example a warning comes?

Reply:
It comes from usage `<Breadcrumb.Divider><Icon /></Breadcrumb.Divider>` and having `content: '/'` as `defaultProps`

Reply:
In this case please change it to `Breadcrumb.defaultProps.children`

Reply:
Please, check it now